packages/sdk/src/actor/physics/physicsBridge.ts

```diff
@@ -4,10 +4,24 @@
  */
 
 import { Guid } from "../../util";
+import { TransformLike } from "..";
 
 export type PhysicsBridgeTransformUpdate = {
     id: Guid;
     time: number;
     transformCount: number;
+    flags: number;
     transformsBlob: string;
 }
+
+export type PhysicsServerOneTransformsUpdate = {
```

Comment:
>PhysicsServerOneTransformsUpdate [](start = 12, length = 32)

This feels like a slightly awkward name.  Would `PhysicsServerTransformUpdate` be better and more concise?

Reply:
Changed from PhysicsServerOneTransformsUpdate to PhysicsServerOneTransformUpdate

---
In reply to: [450331004](https://github.com/microsoft/mixed-reality-extension-sdk/pull/652#discussion_r450331004) [](ancestors = 450331004)